templates/search_derived/search_clients_daily_v8/query.sql

```diff
@@ -52,25 +35,25 @@ WITH
       END
         AS type
       FROM
-        UNNEST(search_counts.list) ),
+        UNNEST(search_counts) element ),
```

Comment:
Aliases need to explicitly use `AS`
```suggestion
        UNNEST(search_counts) AS element),
```